### ISSUE TITLE
Document the differences in performance to your other projects

### ISSUE THREAD

**Issue by davidwynter** [AI4Finance-LLC/Elegant-FinRL#1]
I have been using Deep-Reinforcement-Learning-for-Automated-Stock-Trading-Ensemble-Strategy-ICAIF-2020 to experiment with.

It would be helpful to have documentation on how to alter the FinanceMultiStockEnv to change the labels used from macd, rsi etc to the ones the researcher creates. Further documentation could also show how to conduct the full train, validation and trade cycle as seen in the earlier frameworks.

It is not clear what performance differences there are between that project, FinRL and Elegant-FinRL. A switch from Tensorflow 1.14.1 to Pytorch will introduce changes as will your architecture changes. Can you document the differences in performance on the DJI dataset?

**Comment by bayareacoder** [comment]
I agree with @davidwynter that the lack of documentation about the differences between all these related libraries is very unclear. If there is 1 library that supersedes other versions then that should be stated in the README of the others.
